modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewTest.java

```diff
@@ -1977,6 +2049,47 @@ public void testEventIndicesOnSelectRange() {
         assertEquals("List item at index 2 should be selected", 2, (int) sm.getSelectedIndices().get(1));
     }
 
+    @Test public void testExcludeKeyMappingsForComboBoxEditor() {
+        ListView<String> listView = new ListView<>(FXCollections
+                .observableArrayList("Item1", "Item2"));
+        listView.setCellFactory(TextFieldListCell.forListView());
```

Comment:
hmm .. why the textFieldListCell? It's just a plain listCell if not in editing state .. or not?

Reply:
It is effect of copy paste ;), Test behaves as expected even without it, so removed.